composer.json

```diff
@@ -11,8 +11,9 @@
   },
   "config": {
     "sort-packages": true,
+    "process-timeout": 0,
```

Comment:
This is set to 300 seconds or 5 minutes by default. I think we should keep it at that in order to avoid long running CI tasks. All cases with linters or tests running for more than 5 minutes should probably be investigated and made to run faster. Keeping the limit in place will ensure that we notice when this happens.

Reply:
In the past it's caused problems with the `watch` task, since it's designed to run for hours, but I'll test to make sure that's still the case, and if there's another way around it.

Reply:
I was able to fix it like so 466370bc928fffa71e6d80f0009cb0ee0ce75bb2. Does that solve it?